starwars/src/App.js

```diff
@@ -11,7 +13,10 @@ const App = () => {
 
   return (
     <div className="App">
+      
       <h1 className="Header">React Wars</h1>
+      
```

Comment:
Beware of white space on your page, the readability can suffer from too much white space!